### PR TITLE
Fix/docs typos

### PR DESCRIPTION
**Pull request by samueldg**
## Description

Minor improvements in the User Guide and FAQ.

The FAQ section that changed used to look like:

before

and after the changes:

after

The rest consists of fixed typos.

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -230,8 +230,8 @@ Transaction Model and Locking`_ in the MySQL user manual.
 The worker isn't doing anything, just hanging
 ---------------------------------------------
 
-**Answer:** See `MySQL is throwing deadlock errors, what can I do?`_.
-            or `Why is Task.delay/apply\* just hanging?`.
+**Answer:** See `MySQL is throwing deadlock errors, what can I do?`_,
+or `Why is Task.delay/apply\*/the worker just hanging?`_.
```
Comment (georgepsarakis) [inline docs/faq.rst]: I am not sure about this change, could you please explain the rephrasing?

Comment (samueldg) [inline docs/faq.rst]: This is a reference to another question, which is not aligned with the actual question.

If you look two questions below this answer, you'll see it is indeed:

> Why is Task.delay/apply*/the worker just hanging?

and not 

> Why is Task.delay/apply* just hanging?

**Comment by georgepsarakis** [comment]
Thank you! Feel free to add yourself in the list of [contributors](https://github.com/celery/celery/blob/master/CONTRIBUTORS.txt).

**Comment by samueldg** [comment]
@georgepsarakis Thanks for the review. I updated the contributors list (17d528d), and fixed another typo I found in the meanwhile (fd140e9e).

**Pull request merged**